EIPS/eip-6617.md

```diff
@@ -40,32 +40,94 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 
 *Note* The following specifications use syntax from Solidity `0.8.7` (or above)
 
+```solidity
+pragma solidity ^0.8.7;
+
+/**
+    @title EIP-6617 Bit Based Permission
+    @dev See https://eips.ethereum.org/EIPS/eip-6617
+*/
+library EIP6617 {
+        
+	/**
```

Comment:
Something is possible wrong here, please try the Solidity plugin and perform auto format